nf_research_tools.rdb.model.csv

```diff
@@ -55,15 +55,15 @@ Tumor Type,"The types of tumors that are banked, if applicable.","cutaneous neur
 Disease Type,"The diseases of the banked specimens, if applicable.","Neurofibromatosis type 1, Neurofibromatosis type 2, Schwannomatosis",,TRUE,,Biobank,,,list
 Specimen Preparation Method,The preservation methods used by the biobank.,"Fresh collected, Flash frozen, FFPE, Cryopreserved, OCT, RNA later, Formalin-fixed, Ethanol",,TRUE,,Biobank,,,list
 Specimen Format,How the specimens have been processed in preparation for distribution.,"whole tumor, slide, whole cell lysate, DNA, RNA",,TRUE,,Biobank,,,list
-Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Reference, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Reference, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",
+Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Species, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Species, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",
 Observation_id,A unique identifier for the observation.,,,TRUE,,Observation,,,
 Observation Submitter Name,The name of the person submitting the observation. ,,,TRUE,,Observation,,,
 Synapse ID,The synapse identifier for the person submitting the observation. ,,,FALSE,,Observation,,,
 Observation Text,Free text observation about the resource.,,,TRUE,,Observation,,,
-Observation Type,Type of observation,"pathologic, behavioral-non pathologic, issue, neurochemical, Depositor Comment, Usage Instructions, general comment/review, other",,TRUE,,Observation,,,
-Observation Reference,What t=0 for the observationTime is,"tumor implantation, gestational day zero, birth, immortalization",,FALSE,,Observation,,,
+Observation Type,Type of observation,"Body Length, Body weight, Coat Color, Disease Susceptibility, Feed Intake, Feeding Behavior, Growth rate, Motor Activity, Organ Development, Reflex Development, Reproductive Behavior, Social Behavior, Swimming Behavior, Tumor Growth, Other",,TRUE,,Observation,,,

```

Comment:
```suggestion
Observation Type,Type of observation,"Body Length, Body weight, Coat Color, Disease Susceptibility, Feed Intake, Feeding Behavior, Growth rate, Motor Activity, Organ Development, Reflex Development, Reproductive Behavior, Social Behavior, Swimming Behavior, Tumor Growth, Issue, Depositor Comment, Usage Instructions, General Comment or Review, Other",,TRUE,,Observation,,,

```

Reply:
I think we should still leave some of these in as they represent categories that we have in the data that are not captured by the new terms